source/_patterns/02-organisms/components/content-aside~complete.json

```diff
@@ -110,6 +110,10 @@
           }
         }
       }
-    ]
+    ],
+    "seeMoreLink": {
+      "name": "Further reading",
+      "url": "#"
```

Comment:
Where will this URL come from?

Reply:
Often in Pattern library we are not using real links.